### ISSUE TITLE
 'Prime' Orders Stream Cache

### ISSUE THREAD

**Issue by liampauling** [liampauling/betfair#289]
A current restriction in the order stream is the unavailability of matched orders (Execution Complete) within a market, the stream only contains unmatched orders (UO):

https://docs.developer.betfair.com/display/1smk3cen4v3lu3yomq5qye0ni/Exchange+Stream+API#ExchangeStreamAPI-OrderSubscriptionMessage

This can make things tricky when starting an application on markets with executable orders as it requires a manual currentOrders request, a work around could be to implement a `prime()` function to call currentOrders and prime the cache with all orders before starting the stream.

```python
current_orders = trading.betting.list_current_orders(
    market_ids=["1.165980506"],
)

listener = StreamListener()
listener.prime(current_orders)
```

`prime()` would create the required caches or update if required, it would then output all orders (EC and E)

**Comment by liampauling** [comment]
Out of scope for the library, flumine will have an implementation of this.
